### ISSUE TITLE
Customizable subtitle outline

### ISSUE THREAD

**Issue by Corrah** [Stremio/stremio-features#828]
**Stremio Version**
Stremio 1.6.9 Android TV

**Is your feature request related to a problem? Please describe.**
There is an option available to putting the outline black with 100% opacity but, when watching anime especially, tends to blend in.

**Describe the solution you'd like**
If it possible a feature can be added so the outline can be made more darker?
Or a way to change the type of outline: shading, raised, recessed, and so on
So the subtitle would be more enjoyable

**Comment by TRtomasz** [comment]
Not sure i understand your request, how can be something more dark than black?

**Comment by Corrah** [comment]
> Not sure i understand your request, how can be something more dark than black?

Sorry, it's just difficult to explain. 
Some programs that have subtitles give you the option to choose the edge type. 
For example, the shield has the internal option (not on external apps) of edge types like outline, drop shadow, raised, depressed. On the other hand, VLC has outline thickness options such as thin, normal, thick. 
I wanted to ask if it was possible to have a similar feature on Stremio as well. 
Many times the subtitles tend to blend with the background when it's very light. 
I'll leave some images to make it clearer. 

![Example 1](https://github.com/Stremio/stremio-features/assets/167757382/772499cd-e538-4d11-b703-d4fe034e5e98)
![Example 2](https://github.com/Stremio/stremio-features/assets/167757382/b796e164-fe53-4d8f-9cd9-18fd3c7487a0)

They don't look good, but it changes a lot in person, especially with white backgrounds on a TV.

**Comment by TRtomasz** [comment]
There is such option in stremio, check the Settings -> playback Subtitles Color/Background Color/Outline color.

**Comment by Corrah** [comment]
> There is such option in stremio, check the Settings -> playback Subtitles Color/Background Color/Outline color.

Yes, but as I showed you with all the possible imaginary options, even putting text in bold and everything, the borders are very subtle as in the first example.
In the second one, which is an external app, the subtitles have a much more pronounced border, making it more readable

**Comment by quincyp** [comment]
To chime in here:
I believe the option you are suggesting is an option to increase outline size/thickness and I agree this would be a nice addition

**Comment by ItsRogueRen** [comment]
> There is such option in stremio, check the Settings -> playback Subtitles Color/Background Color/Outline color.

I'm also chiming in, the outline on subtitles is too thin when watching animated shows/movies. Animation has more solid colors then live action shows so it is significantly easier for the subtitles to blend in with the content, to the point that on a TV far away from you it can become so thin that the outline is practically non-existent. An option to have much thicker outlines (such as the style used on Crunchyroll for anime) would dramatically increase legibility for the TV interface where the screen tends to be much further away.

**Comment by Rebbygalaxy** [comment]
I'd like add to this, I have subtitles where the outline is so thing it might as well not exist, which can make it very hard to read. Having an option to control how thick the outline is would massively improve legibility for subtitles.

<img width="1772" height="998" alt="Image" src="https://github.com/user-attachments/assets/e1799916-e279-47c3-900f-47ea52c6ac08" />

**Comment by procobain** [comment]
Please, when you can, help us with this; 
I'm posting another example below:

<img width="1920" height="3240" alt="Image" src="https://github.com/user-attachments/assets/f0b47ed9-ad0f-4d85-98a6-5be8b5281bf0" />

**Comment by qorrymarzuki** [comment]
My wife kinda need this. She can't read well the subs in anime
